guides/common/modules/con_project-server-upgrade-scenarios-and-prerequisites.adoc

```diff
@@ -4,12 +4,16 @@
 This section describes how to upgrade {ProjectServer} from {ProjectVersionPrevious} to {ProjectVersion}.
 You can upgrade from any minor version of {ProjectServer} {ProjectVersionPrevious}.
 
-include::upgrading_satellite_server_prerequisites.adoc[leveloffset=+3]
+include::snip_upgrading-project-server-prerequisites.adoc[leveloffset=+3]
 
 ifdef::satellite[]
```

Comment:
Hi @maximiliankolb I think we can skip lines 9-16, since they just contain hyperlink to the very next chapter. And yes, 18+19 should belong to connected. I'll put that under the `ifeval::["{mode}" == "connected"]` conditional. Does that sound good?

Reply:
Yes, I think so. I am currently unsure how "mode" is/can be defined.